kudos.yaml

```diff
@@ -2826,8 +2827,6 @@
   image: 90_games.svg
 - name: let's play some music
   to_address: '0xa6d91F3795f934bb5f703724337f39C0f1a95fD5'
-  image: Cassette.svg
```

Comment:
did you mean to delete these lines?

Reply:
In my file, these parameters were duplicated, so I thought it would be correct to fix it.
https://github.com/GuistF/kudosbadges/commit/8e67e51f6471885dd71f719f24b37b1d9d062dbe